CHANGELOG.unreleased.md

```diff
@@ -20,6 +20,7 @@
 - [S3] Fix S3 remote with empty directory not showing anything to restore (PR [#6218](https://github.com/vatesfr/xen-orchestra/pull/6218))
 - [S3] remote fom did not save the `https` and `allow unatuhorized`during remote creation (PR [#6219](https://github.com/vatesfr/xen-orchestra/pull/6219))
 - [VM/advanced] Fix various errors when adding ACLs [#6213](https://github.com/vatesfr/xen-orchestra/issues/6213) (PR [#6230](https://github.com/vatesfr/xen-orchestra/pull/6230))
+- [Remote] show more details when remote testing fails (PR [#6216](https://github.com/vatesfr/xen-orchestra/pull/6216))
```

Comment:
object storage on ceph can generate error messages that the aws sdk can not understand, thus resulting in cryptic messages for the user. 
This PR try to show a usable message in XO and add the full error in the logs

It comes from the support ticket 6738

Reply:
Ok, then maybe something like this:

```suggestion
- [Remote] Show complete errors instead of only a potentially missing message (PR [#6216](https://github.com/vatesfr/xen-orchestra/pull/6216))
```